scripts/download_incr_DB.py

```diff
@@ -137,10 +137,23 @@ def GetAllObjectsFromS3(url, folderName=""):
 	print("[" + str(datetime.datetime.now()) + "]"+" All objects from " + url + " completed!")
 	return 0
 
+def CheckRetryLimit(retry_counter,filename):
+	if retry_counter > 3:
+		print("Giving up after " + str(retry_counter) + " retries for file: " + filename + " ! Please check with support team.")
+		time.sleep(5)
+		os._exit(1)
```

Comment:
better not to exit. just give up and continue with other files. It will help us debug in case.
Also, Instead return 0 or 1 and check at line 156 to decide if to `continue` or `return`

`CheckRetryLimit` looks more of overkill in this case. Better to check retry limit in place.